tests/lib/rules/no-redeclare.js

```diff
@@ -105,8 +105,8 @@ ruleTester.run("no-redeclare", rule, {
             options: [{ builtinGlobals: true }],
             env: { browser: false }
         },
-        { code: "var glovalThis = foo", options: [{ builtinGlobals: true }], env: { es6: true } },
-        { code: "var glovalThis = foo", options: [{ builtinGlobals: true }], env: { es2017: true } },
+        { code: "var globalThis = foo", options: [{ builtinGlobals: true }], env: { es6: true } },
```

Comment:
Not sure if this was testing a specific case

Reply:
This is to test that declarations of `globalThis` global variable aren't reported in targeted environments where `globalThis` is not a built-in global variable and thus a declaration in the code is not a redeclaration.

The change you made is good. With the typo, these two tests were not testing what they were supposed to.